### ISSUE TITLE
question: controller-as-vm options

### ISSUE THREAD

**Issue by glittle1972** [EmmanuelDemey/eslint-plugin-angular#585]
Apologies if raising an issue is not the correct way to find this out.

In my code I have used "self" as the variable name in the ControllerAs convention as this was used in most of the online examples and tutorials I found. As many tutorials seem to use this naming convention I would like to continue using it.

```
var self = this;
self.reason = undefined;
```

How can I configure angular/controller-as-vm to use "self" as the consistent variable name? This is what I have added to eslintrc.js so far (I have also tried a number of similar variations)

```
"rules": {
  "angular/controller-as-vm": [0, "self"]
}
```

but as soon as I add anything after angular/controller-as-vm, no warning is raised (even if I use other values than "self")

Thank you in advance, Gordon

**Comment by glittle1972** [comment]
Never mind. I just tried something and discovered that 0 = off, 1 = warning, 2 = error. Apologies for messing up your issue list.
